docs/api/callbacks.rst

```diff
@@ -0,0 +1,14 @@
+****************
+Choice Functions
```

Comment:
Why is the title `Choice Functions` here?

Reply:
Added in the next commit